android/src/main/res/base/anim/rns_slide_in_from_right_ios.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<translate xmlns:android="http://schemas.android.com/apk/res/android"
+    android:duration="@android:integer/config_shortAnimTime"
+    android:interpolator="@android:interpolator/accelerate_decelerate"
```

Comment:
One more thing - I've also found that interpolator is only provided here. Is it needed?

Reply:
Yeah basically it tries to somewhat resemble iOS only for `newTop` fragment. Dismissing and bottom fragments do not have the animation (Or I couldn't notice)

Reply:
I understand, but doesn't we also require it in `rns_slide_in_from_left_ios` then?

Reply:
Added. It looks better now